lib/osf-components/addon/components/paginated-list/layout/template.hbs

```diff
@@ -50,14 +50,28 @@
         </ul>
     {{/if}}
     {{#if this.paginatorShown}}
-        <div local-class='text-center'>
-            {{simple-paginator
-                maxPage=this.maxPage
-                nextPage=(action @next)
-                previousPage=(action @previous)
-                curPage=@page
-            }}
-        </div>
+        {{#if (eq this.pagination 'adjustable')}}
+            <div local-class='text-center'>
+                {{adjustable-paginator
+                    maxPage=this.maxPage
+                    nextPage=(action @next)
+                    previousPage=(action @previous)
+                    page=@page
+                    doReload=@doReload
+                    pageSize=@pageSize
+                    totalCount=@totalCount
+                }}
+            </div>
+        {{else}}
+            <div local-class='text-center'>
+                {{simple-paginator
+                    maxPage=this.maxPage
+                    nextPage=(action @next)
+                    previousPage=(action @previous)
+                    curPage=@page
+                }}
```

Comment:
I know this is what was there before but can we invoke this component using `<AngleBracket>` invocation like..
```
                <AdjustablePaginator
                    @maxPage={{this.maxPage}}
                    @nextPage={{action @next}}
                    //... the rest of the args
                />
            </div>
        {{else}}
            <div local-class='text-center'>
                <SimplePaginator
                    @maxPage={{this.maxPage}}
                    // ... the rest of the args like this too
                />
```